designsafe/static/scripts/data-depot/components/data-depot-listing/files-listing.template.html

```diff
@@ -22,7 +22,7 @@
                     <th>Size</th>
                     <th>Last modified</th>
                     <th ng-if="$ctrl.operationLabel">Operation</th>
-                    <th ng-if="!$ctrl.FileListingService.listings.main.loading && $ctrl.hasNotebooks()">Open with</th>
+                    <!-- <th ng-if="!$ctrl.FileListingService.listings.main.loading && $ctrl.hasNotebooks()">Open with</th> -->
```

Comment:
Let's just cut these out. We can always check the repo history if we need to add them back for any reason.
```suggestion
```